js-testing.test.js

```diff
@@ -11,20 +11,30 @@
 
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
 
+// describe("addThemUp", () =>{
+//     test ("takes two numbers as arguments and returns the sum", () =>{
+//         expect(addThemUp(1, 2)).toEqual(1 + 2)
+//     })
+// })  
 
+// // // --------------------1b) See the test fail. THEN write the code to make the test pass.
 
-
-
-// --------------------1b) See the test fail. THEN write the code to make the test pass.
-
-
+// const addThemUp = (num1, num2) => {
+//     return num1 + num2
+// }
 
 
 
 // --------------------2a) Create a test for a function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two.
 
-
-
+describe("triangleArea", () =>{
+    test ("takes the base times the height divided by two to return the area", () =>{
+        expect(triangleArea(6, 10)).toEqual(6 * 10 / 2)
```

Comment:
Your `.toEqual` _can_ be arithmetic like this, but the total return value would be more effective for testing